client/notifications/index.jsx

```diff
@@ -174,7 +173,7 @@ export class Notifications extends Component {
 		const customMiddleware = {
 			APP_RENDER_NOTES: [
 				( store, { newNoteCount } ) => {
-					this.props.setIndicator( newNoteCount );
+					localStorageHelper.set( 'wpnotes_unseen_count', newNoteCount );
```

Comment:
Do we even need this? Im not seeing a `.get` for it so it seems like it is unused.

Beyond that, we are managing the unseenCount in state. And Im a bit confused how localStorage would be useful for storing the notes count since the data would be stale as soon as the session ends (could have changed your notes count from interacting with it outside of calypso, site front end, atomic, etc.) 😕

Reply:
Yep, looks like we can remove it 🙂